exporter/awsemfexporter/datapoint.go

```diff
@@ -110,7 +110,16 @@ type summaryMetricEntry struct {
 // At retrieves the NumberDataPoint at the given index and performs rate/delta calculation if necessary.
 func (dps numberDataPointSlice) At(i int) (dataPoint, bool) {
 	metric := dps.NumberDataPointSlice.At(i)
-	labels := createLabels(metric.LabelsMap(), dps.instrumentationLibraryName)
+	// To work backward compatible on the old OTel SDKs
+	// will get metric dimensions from Attributes first
+	// then get the dimension from LabelsMap if Attributes is not populated
+	attributes := make(map[string]string)
+	if metric.Attributes().Len() > 0 {
+		attributes = createAttributes(metric.Attributes(), dps.instrumentationLibraryName)
+	} else {
+		attributes = createLabels(metric.LabelsMap(), dps.instrumentationLibraryName)
```

Comment:
You don't need to do this. We do the conversion on the source, so no need to check for labels.

Reply:
Ah. Nice thing that I missed. I'll revert this part. Ty!